docs/src/data/frameworks.ts

```diff
@@ -0,0 +1,3 @@
+export type Frameworks = string[];
+
+export const FRAMEWORKS: Frameworks = ['react', 'angular', 'vue', 'flutter'];
```

Comment:
I'll make another PR to use `FRAMEWORKS` to refactor some other parts of our codebase

Reply:
Thanks for adding this